src/lib/logger/logger.ts

```diff
@@ -120,18 +111,28 @@ export function setup(options: LoggerConfig | LoggerConfigItem = [DEFAULT_LOGGER
     );
   }
   const pinoConfig = { level: loggerConfig.level };
+  let colors = 'colors' in loggerConfig ? loggerConfig.colors : process.stdout.isTTY;
+  if ('EXPERIMENTAL__VERDACCIO_LOGGER_COLORS' in process.env) {
```

Comment:
Sorry was my typo maybe you remove the double _ . My bad

Reply:
oh, I thouhgt it was interntional... :P 

roger.